cli/README.md

```diff
@@ -69,3 +70,51 @@ Global Flags:
 -kubeconfig, -c=<string>
 	Path to kubeconfig file
 ```
+
+### consul-k8s uninstall
+This command uninstalls Consul on Kubernetes, while prompting whether to uninstall the release and whether to delete all
+related resources such as PVCs, Secrets, and ServiceAccounts.
+
+Get started with:
+```bash
+consul-k8s uninstall
+```
+
+```
+Usage: kubectl consul uninstall [options]
+
+Uninstall Consul and clean up all data.
+Any data store in Consul will not be recoverable.
+Command Options:
+
+  -auto-approve
+      Skip approval prompt for uninstalling Consul. The default is false.
+
+  -name=<string>
+      Name of the installation. This will be prefixed to resources installed
+      on the cluster.
```

Comment:
I'm pretty sure this is a copy/pasta issue, but I'm not sure why we need the second sentence in the context of uninstall? :)

Reply:
Yeah good point! Updated that sentence to read: "This can be used to uninstall and/or delete the resources of a specific Helm release."